.github/workflows/test_gui.yml

```
@@ -412,6 +412,26 @@ jobs:
           Invoke-WebRequest -Uri 'https://go.microsoft.com/fwlink/p/?LinkId=2124703' -OutFile 'setup.exe'
           Start-Process -FilePath setup.exe -Verb RunAs -Wait
 
+      # tauri-driver drives the app through msedgedriver, which must match
+      # the WebView2 Runtime version to establish a session at all
+      - name: Install matching Edge WebDriver (Windows)
+        if: matrix.os == 'windows-latest'
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC1036:error:1:12: '(' is invalid here. Did you forget to escape it?

(shellcheck)